common/src/main/java/gov/cms/ab2d/common/util/FileUtil.java

```diff
@@ -0,0 +1,35 @@
+package gov.cms.ab2d.common.util;
```

Comment:
We should keep the code used for testing under `test`, not `main`.

Reply:
I had tried putting it in the test project under common, but I don't think it's visible to the code that uses it in api if it's under there.